docs/data/base/components/input/UnstyledInputIntroduction/tailwind/index.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { Input } from '@mui/base/Input';
+import { useTheme } from '@mui/system';
+
+function useIsDarkMode() {
+  const theme = useTheme();
+  return theme.palette.mode === 'dark';
+}
+
+export default function UnstyledInputBasic() {
+  // Replace this with your app logic for determining dark modes
+  const isDarkMode = useIsDarkMode();
+
+  return (
+    <Input
+      className={isDarkMode ? 'dark' : ''}
+      slotProps={{
+        input: {
+          className:
+            'w-80 text-sm font-normal leading-5 px-3 py-2 rounded-lg shadow-md shadow-slate-100 dark:shadow-slate-900 focus:shadow-outline-purple dark:focus:shadow-outline-purple focus:shadow-lg border border-solid border-slate-300 hover:border-purple-500 dark:hover:border-purple-500 focus:border-purple-500 dark:focus:border-purple-500 dark:border-slate-600 bg-white dark:bg-slate-900 text-slate-900 dark:text-slate-300 focus-visible:outline-0',
```

Comment:
It looks like we missed the configuration of the font family in the `tailwind.config.js` file.

I imagine we should use Inter to match Joy UI?

**Off-topic** I found a broader bug with font 🙃, reported in #38306.

Reply:
Opened https://github.com/mui/material-ui/pull/38464 for the font family.